packages/apps/docs/src/docs/marmalade/what-is-marmalade.md

```diff
@@ -0,0 +1,180 @@
+
+With this separation of duties, the `sale` pact is responsible for transferring the NFT out of escrow and enforcing the timeout.
+Any other transaction requirements are handled by token policies.
+
+### Rollback and withdraw
+
+The timeout captured in the **offer** step allows a seller to withdraw from the sale if there are no interested buyers for the token. 
+The timeout is measured in blocks, such that if a seller sets the timeout to 30 blocks, he can't pull out of the sale until 30 blocks have been mined on-chain.
+
+After the timeout, the seller can send a **rollback** continuation message for the identifier assigned to the SALE event. 
+The **withdraw** operation then transfers the token out of the escrow account and returns it to the seller.
+
+### Completing a logical sequence
+
+All of the logic required to step through the sales process is automatically handled by the Pact language.
+The pact definition ensures that step two can only happen after step one and only under the following conditions:
+
+- If step one hasn't been rolled back.
+- If step two hasn't already happened.
+
+Similarly, a rollback can’t occur if step one hasn’t happened, or if step two has already happened. 
+With Pact, this logic isn't left to contract programmers.
+Instead, Pact prevents the contract developer from introducing faulty logic in sequencing the transaction.
+
+## Using built-in policies
+
+The Marmalade token standard defines interfaces for policy-based token management.
+To simplify token creation using the standard, Kadena provides several built-in token policies for the most common features.
+You can use the built-in policies—called **concrete policies**—independently or in combination to provide the specific features you want to implement.
+For example:
+
+- Use the **guard policy** to prevent unauthorized token activity like minting or burning.
+- Use the **non-fungible policy** to ensure that a token can only be minted once and never burned.
+- Use the **royalty policy** to specify the account for depositing royalty payments from sales.
+- Use the **collection policy** to create token collections for a marketplace.
+
+To illustrate how you can use these policies, assume that you want to offer a non-fungible token.
+You want to ensure that you are the only person authorized to mint the token and that only one token can be minted and owned.
+You also want to require that the token can only be transferred to a new owner through a sale contract and you'll be paid a royalty any time the token ownership changes.
+
+To accomplish this, you can create the token using the `marmalade-v2.guard-policy-v1`,`marmalade-v2.non-fungible-policy-v1`, and `marmalade-v2.royalty-policy-v1` policies.
+
+These policies ensure that you are the only person authorized to mint the token, only one token can be minted, and any time the token is transferred to a new owner, you're paid the  royalty you specify.
```

Comment:
To be picky, adding guard-policy doesn't automatically add the mint-guard, but it requires the token creator to add "mint-guard" information at creation.